wappsto/connection/communication.py

```diff
@@ -618,19 +619,24 @@ def receive_data(self):
         decoded = None
         while True:
             if self.connected:
-                data = self.my_socket.recv(2000)
+                data = self.my_socket.recv(MESSAGE_SIZE)
                 decoded_data = data.decode('utf-8')
```

Comment:
You should add ignore to the decode, to make sure that there is no exceptions thrown.